pkg/rid/store/store.go

```diff
@@ -16,7 +16,7 @@ type Store interface {
 	Transactor
 
 	// Get store version
-	GetVersion(ctx context.Context) (*semver.Version, error)
+	GetVersion() semver.Version
```

Comment:
I don't think we should adjust the interface and instead make the caching optimization an implementation detail. After all, an implementation talking to the backend data store would certainly be reasonable.

Reply:
Sorry, I'm not quite get it. It doesn't hurt to have this optimization. Is there a chance that db version is changed in the fly? Does db migration process happens only when we restart server?

Reply:
What I think @vosst might mean is, that we are change the way we check the version when we don't really need to. I think he is suggesting we keep the function signature the same. What we can do is have GetVersion still try to reach the backend and simply store the version when it gets it for the first time, and on subsequent gets check if its already cached the information then return cache if it exists.

Reply:
oh. Got it. Thanks :)